doc/developer/develop.md

```diff
@@ -105,7 +105,7 @@ Materialize is fully integrated with Cargo, so building it is dead simple:
 ```shell
 git clone git@github.com:MaterializeInc/materialize.git
 cd materialize
-cargo run --bin materialized
+cargo run --bin materialized --release -- --threads #
```

Comment:
I think a possibly-better doc is:

```shell
cargo run --release -- --threads 2
```

The `default-run` parameter is set in `src/materialized/Cargo.toml` which I just verified is working correctly, and we might as well make the command copy-pastable.

Reply:
Leaving `--bin materialized` in as a hint that you can run other commands seems fine, too, though.